### ISSUE TITLE
Analytics Graph is not accessible by screen readers.

### ISSUE THREAD

**Issue by njaffer** [samvera/hyrax#4541]
### Descriptive summary
Analytics Graph is not accessible by screen readers. The content/controls in the Work Analytics are not accessible to a screenreader or keyboard user.

### Expected behavior
Analytics Graph is accessible by screen readers. 

### Actual behavior
When the analytics page is read by screen readers, the user gets the analytics data. A short term solution could be  achieved by adding a complementary text table with with data points in addition to the graph.

### Steps to reproduce the behavior
1. Visit Analytics page from Works details page
2. Use screen reader to read the analytics data
3. Use tabs to go through the graph

### Related work

Link to related tickets or prior related work here.
https://github.com/samvera/hyrax/issues/4530

**Comment by jlhardes** [comment]
Work analytics are disabled on all nurax systems at this point so this issue cannot be evaluated right now. Will leave this open if we are able to enable analytics and test with a screen reader/keyboard in the future.
